lemmy_utils/src/email.rs

```diff
@@ -18,35 +20,63 @@ pub fn send_email(
   html: &str,
 ) -> Result<(), String> {
   let email_config = Settings::get().email.ok_or("no_email_setup")?;
+  let domain = &Settings::get().hostname;
 
-  let email = Email::builder()
-    .to((to_email, to_username))
-    .from(email_config.smtp_from_address.to_owned())
+  let email = Message::builder()
+    .from(
+      email_config
+        .smtp_from_address
+        .parse()
+        .expect("from address isn't valid"),
+    )
+    .to(Mailbox::new(
+      Some(to_username.to_string()),
+      Address::from_str(to_email).expect("to address isn't valid"),
+    ))
     .subject(subject)
-    .html(html)
-    .build()
-    .unwrap();
-
-  let mailer = if email_config.use_tls {
-    SmtpClient::new_simple(&email_config.smtp_server).unwrap()
-  } else {
-    SmtpClient::new(&email_config.smtp_server, ClientSecurity::None).unwrap()
+    .multipart(
+      MultiPart::mixed().multipart(
+        MultiPart::alternative()
+          .singlepart(
+            SinglePart::eight_bit()
+              .header(header::ContentType(
+                "text/plain; charset=utf8".parse().unwrap(),
+              ))
+              .body(html),
+          )
+          .multipart(
+            MultiPart::related().singlepart(
+              SinglePart::eight_bit()
+                .header(header::ContentType(
+                  "text/html; charset=utf8".parse().unwrap(),
```

Comment:
Remove unwrap (also in line 43).

Reply:
Well it might be okay like this cause its just parsing a static string, but cleaner with unwrap.